rush.json

```diff
@@ -141,8 +141,13 @@
       "shouldPublish": false
     },
     {
-      "packageName": "office-ui-fabric-react-tslint",
-      "projectFolder": "packages/office-ui-fabric-react-tslint",
+      "packageName": "@uifabric/tslint-rules",
+      "projectFolder": "packages/tslint-rules",
+      "shouldPublish": false
+    },
```

Comment:
I'm surprised at shouldPublish: false. Is this something you plan to flip after manual publish?